### ISSUE TITLE
Game crashed

### ISSUE THREAD

**Issue by THEALPACA** [btwonion/SimpleAutoDrop#5]
### What Happened

Game crashed on game launching stage(the red background with mojang logo), with silk 1.9.2, fabric api 0.81.1 and fabric language kotlin 1.9.4 installed, game version`1.19.4`.

### Debug Information

```

Description: Unexpected error

java.lang.InstantiationError: net.minecraft.class_7157
	at net.minecraft.class_634.handler$bfi000$silk-commands$onLogin(class_634.java:7706)
	at net.minecraft.class_634.method_11120(class_634.java:463)
	at net.minecraft.class_2678.method_11567(class_2678.java:92)
	at net.minecraft.class_2678.method_11054(class_2678.java:25)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)
	at net.minecraft.class_1255.method_5383(class_1255.java:115)
	at net.minecraft.class_310.method_1523(class_310.java:1174)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_634.handler$bfi000$silk-commands$onLogin(class_634.java:7706)
	at net.minecraft.class_634.method_11120(class_634.java:463)
	at net.minecraft.class_2678.method_11567(class_2678.java:92)
	at net.minecraft.class_2678.method_11054(class_2678.java:25)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)

-- Affected level --
Details:
	All players: 1 total; [class_746['*'/0, l='ClientLevel', x=8.50, y=65.00, z=8.50]]
	Chunk stats: 256, 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Section: (at 8,0,8 in 0,4,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Server brand: ~~NULL~~
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:455)
	at net.minecraft.class_310.method_1587(class_310.java:2406)
	at net.minecraft.class_310.method_1514(class_310.java:825)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

```

### Other Details

```
Fabric Mods: 
		autoclicker-fabric: Auto Clicker 1.19.4-1.6.1-build.22
		autodrop: SimpleAutoDrop 1.5.2-1.19.4
		baritone: Baritone 1.9.3
		cem: Custom Entity Models 0.7.3
			completeconfig-base: completeconfig-base 2.3.1
			completeconfig-gui-cloth: completeconfig-gui-cloth 2.3.1
		controlling: Controlling For Fabric 11.1.4
		fabric-api: Fabric API 0.81.1+1.19.4
			fabric-api-base: Fabric API Base 0.4.26+1e9487d2f4
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.27+504944c8f4
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.8+348a9c64f4
			fabric-block-api-v1: Fabric Block API (v1) 1.0.7+e022e5d1f4
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.36+c2e6f674f4
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.17+504944c8f4
			fabric-command-api-v1: Fabric Command API (v1) 1.2.29+f71b366ff4
			fabric-command-api-v2: Fabric Command API (v2) 2.2.8+504944c8f4
			fabric-commands-v0: Fabric Commands (v0) 0.2.46+df3654b3f4
			fabric-containers-v0: Fabric Containers (v0) 0.1.56+df3654b3f4
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.11+ae0966baf4
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.1+fe8721bef4
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.16+aeb40ebef4
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.4.2+504944c8f4
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.47+7f87f8faf4
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.15+504944c8f4
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.5.1+76ba65ebf4
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.54+df3654b3f4
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.34+a1ccd7bff4
			fabric-item-api-v1: Fabric Item API (v1) 2.1.19+504944c8f4
			fabric-item-group-api-v1: Fabric Item Group API (v1) 3.0.7+043f9acff4
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.34+504944c8f4
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.32+df3654b3f4
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.17+1e9487d2f4
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.29+75e98211f4
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.33+9e7660c6f4
			fabric-message-api-v1: Fabric Message API (v1) 5.1.3+504944c8f4
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.41+49abcf7ef4
			fabric-models-v0: Fabric Models (v0) 0.3.32+504944c8f4
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.3+504944c8f4
			fabric-networking-v0: Fabric Networking (v0) 0.3.43+df3654b3f4
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 7.0.5+504944c8f4
			fabric-particles-v1: Fabric Particles (v1) 1.0.25+f1e4495bf4
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.10+a1ccd7bff4
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.2.2+504944c8f4
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.2.7+ebc93ff3f4
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.2.0+ebc93ff3f4
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.40+df3654b3f4
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.30+afca2f3ef4
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.23+504944c8f4
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.43+df3654b3f4
			fabric-rendering-v1: Fabric Rendering (v1) 2.1.3+504944c8f4
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.2+e6c7d4eef4
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.4+5ade3c38f4
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.47+3bd4ab0ff4
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.22+504944c8f4
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.10+504944c8f4
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.2.0+80d07a0af4
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 3.1.1+b4a333d6f4
		fabric-language-kotlin: Fabric Language Kotlin 1.9.4+kotlin.1.8.21
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.8.21
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.20.2
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.5.0
		fabricloader: Fabric Loader 0.14.19
		indium: Indium 1.0.15+mc1.19.4
		inventoryprofilesnext: Inventory Profiles Next 1.10.2
		iris: Iris 1.6.1
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre12
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		java: OpenJDK 64-Bit Server VM 17
		jei: Just Enough Items 13.1.0.11
		libipn: libIPN 3.0.1
		litematica: Litematica 0.14.3
		malilib: MaLiLib 0.15.4
		minecraft: Minecraft 1.19.4
		modmenu: Mod Menu 6.2.2
		searchables: Searchables 1.0.3
		silk-all: Silk (All modules) 1.9.2
			silk-commands: Silk Commands 1.9.2
			silk-core: Silk Core 1.9.2
			silk-game: Silk Game 1.9.2
			silk-igui: Silk Inventory GUI 1.9.2
			silk-nbt: Silk NBT 1.9.2
			silk-network: Silk Network 1.9.2
			silk-persistence: Silk Persistence 1.9.2
		sodium: Sodium 0.4.10+build.24
		xaerominimap: Xaero's Minimap 23.4.3
		xaeroworldmap: Xaero's World Map 1.30.2
	Loaded Shaderpack: ComplementaryReimagined_r2.1.zip
		Profile: Custom (+7 options changed by user)

```

**Comment by THEALPACA** [comment]
I have removed silk, but this time game crashed when picking up items and auto-drop enabled, and there is no crash report file in crash-reports folder, game exit code was `-805306369`

**Comment by btwonion** [comment]
can you paste the content of latest.log in here? you can find it in your game directory > logs > latest.log

**Comment by btwonion** [comment]
the most important lines are the last 100-200

**Comment by btwonion** [comment]
in there there is no crash. make sure to send the log after the game crashed. when you launch it the log will be overidden

**Comment by THEALPACA** [comment]
I've tried many times, every time game crashed when picking up items as well as auto drop enabled, except no item was added to archive. There is no crash report and error logs in log file, game exit code was the same `-805306369`, weird.

**Comment by btwonion** [comment]
i will test it out in the following days. are there multiple archives activated? is the error item independent?

**Comment by THEALPACA** [comment]
I only create one archive when test, and it seems any item or group of them will cause crash.

**Comment by btwonion** [comment]
this bug is fixed with #6
